WooCommerce/Classes/Authentication/Store Creation/Profiler/Selling Status/StoreCreationSellingPlatformsQuestionViewModel.swift

```diff
@@ -0,0 +1,159 @@
+import Combine
+import Foundation
+
+/// View model for the second step of `StoreCreationSellingStatusQuestionView`, an optional profiler question about store selling status
+/// in the store creation flow.
+/// When the user previously indicates that they're already selling online, this view model provides data for the followup question on the platforms they're
+/// already selling on.
+@MainActor
+final class StoreCreationSellingPlatformsQuestionViewModel: StoreCreationProfilerQuestionViewModel, ObservableObject {
+    /// Other online platforms that the user might be selling. Source of truth:
+    /// https://github.com/Automattic/woocommerce.com/blob/trunk/themes/woo/start/config/options.json
+    enum Platform: Equatable {
+        case amazon
+        case bigCartel
+        case bigCommerce
+        case eBay
+        case etsy
+        case facebookMarketplace
+        case googleShopping
+        case pinterest
+        case shopify
+        case square
+        case squarespace
+        case wix
+        case wordPress
+    }
+
+    let topHeader: String
+
+    let title: String = Localization.title
+
+    let subtitle: String = Localization.subtitle
+
+    /// Question content.
+    /// TODO: 8376 - update values when API is ready.
+    let platforms: [Platform] = [
```

Comment:
Nit: If we will always use all the cases from `Platform` we could make `Platform` confirm to `CaseIterable` and use `let platforms = Platform.allCases` here.

Reply:
great suggestion! it looks like `allCases` [follows the order of the case declarations](https://developer.apple.com/documentation/swift/caseiterable) so that's good for the use case. updated in https://github.com/woocommerce/woocommerce-ios/pull/8497/commits/153cb15c9d3523aafb800f16a398e18fae7dff94